src/components/elementView/elementView.scss

```diff
@@ -36,13 +36,13 @@
 .inline-buttons {
   display: inline-block;
   white-space: nowrap;
-  overflow: auto;
+  //overflow: auto;
 }
 
 .inline-buttons-user {
   display: inline-block;
   white-space: nowrap;
-  overflow: hidden;
+  //overflow: hidden;
```

Comment:
this fixed tooltip, really hope it didnt fuck shit up! and it did not to the best of my knowledge.